src/memote/experimental/config.py

```diff
@@ -176,3 +183,30 @@ def get_path(self, obj, default):
         if not isabs(path):
             path = join(self._base, path)
         return path
+
+    def get_min_growth(self, model, threshold=0.1):
+        """Calculate min growth default value or return input value.
+
+        This value is used to determine if a model is capable of growth under
+        certain experimental conditions.
+
+        Parameters
+        ----------
+        model : cobra.Model
+        threshold : float
+            If no input is provided by the user the default value is set to a
+            coefficient `threshold` times the growth under default constraints.
+            Default: 0.1.
+
+        """
+        min_growth = self.config.get("min_growth")
+        if min_growth is None:
+            min_growth = model.slim_optimize() * threshold
+            if isnan(min_growth):
+                LOGGER.debug(
```

Comment:
By default pytest will swallow this message but it is a significant error that I care about as a user of memote.

```suggestion
                LOGGER.error(
```